technische-universitat-dresden-forstwissenschaft.csl

```diff
@@ -0,0 +1,428 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="de-DE">
+  <info>
+    <title>Technische Universität Dresden - Forstwissenschaft (author-date, German)</title>
+    <title-short>TU Dresden - Forst</title-short>
+    <id>http://www.zotero.org/styles/technische-universitat-dresden-forstwissenschaft</id>
+    <link href="http://www.zotero.org/styles/technische-universitat-dresden-forstwissenschaft" rel="self"/>
+    <link href="http://www.zotero.org/styles/die-bachelorarbeit-samac-et-al-note" rel="template"/>
+    <link href="http://www.amazon.de/Bachelorarbeit-P%C3%A4dagogischen-Hochschule-wissenschaftlicher-Lehrerbildung/dp/3708901886" rel="documentation"/>
+    <author>
+      <email>jens.mittelbach@slub-dresden.de</email>
+      <uri>https://www.jensmittelbach.de</uri>
+      <name>Jens Mittelbach</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
```

Comment:
fix both the field (engineering might be best?) and the summary please (OK to just delete the summary if you have nothing to add beyond documentation)